utils/base_test.py

```diff
@@ -0,0 +1,29 @@
+import os
+import unittest
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.chrome.service import Service
+
+
+class BaseTest(unittest.TestCase):
+    def _setUp(self, url=None) -> None:
```

Comment:
Why prepend _ to the method name? Is it to make it available from the child classes? If yes, better way to implement it is to keep setUp() name here and use super() in the child's class setUp() to call it - see comment above

Reply:
This is not my code, I don't know. We are all working together so if you have any suggestions - you can just change files and Baur will review it. We all are coding on different levels so the code doesn't suppose to be perfect